luda-editor/new-server/rpc/src/types/scene.rs

```diff
@@ -1,4 +1,5 @@
 use migration::schema::SceneDoc;
+use migration::schema::SceneSound as SceneSound_;
 
 pub type Scene = SceneDoc;
-
+pub type SceneSound = SceneSound_;
```

Comment:
Why not just `pub use`?

Reply:
for unity.`pub type Scene = SceneDoc;`-like pattern.

Reply:
I think so